pixmaps.ml: Error: Unbound module Bytearray

Sorry for not noticing the build failures with b24b600 earlier:

````
....
[   31s] + /usr/bin/make -O -j4 PREFIX=/usr
....
[   31s] make[1]: Entering directory '/home/abuild/rpmbuild/BUILD/unison-20241102T085235.b24b600/src'
[   31s] ocamlopt: pixmaps.ml ---> pixmaps.cmx
[   31s] ocamlopt -g -I lwt -I ubase -I system -I +unix -I +str -I system/generic -I lwt/generic -I +lablgtk3 -I +cairo2 -c /home/abuild/
rpmbuild/BUILD/unison-20241102T085235.b24b600/src/pixmaps.ml
[   31s] File "/home/abuild/rpmbuild/BUILD/unison-20241102T085235.b24b600/src/pixmaps.ml", line 260, characters 12-28:
[   31s] 260 |   let buf = Bytearray.create sz in
[   31s]                   ^^^^^^^^^^^^^^^^
[   31s] Error: Unbound module Bytearray
[   31s] Hint: Did you mean Bigarray?
[   31s] make[1]: *** [Makefile.OCaml:426: pixmaps.cmx] Error 2
[   31s] make[1]: Leaving directory '/home/abuild/rpmbuild/BUILD/unison-20241102T085235.b24b600/src'
````

--- a/src/.depend
+++ b/src/.depend
@@ -761,8 +761,10 @@ path.cmi : \
     ubase/umarshal.cmi \
     pred.cmi \
     name.cmi
-pixmaps.cmo :
-pixmaps.cmx :
+pixmaps.cmo : \
+    bytearray.cmi
+pixmaps.cmx : \
+    bytearray.cmx
 pred.cmo : \
     ubase/util.cmi \
     ubase/umarshal.cmi \

